mkdocs.yaml

```diff
@@ -8,45 +8,72 @@ nav:
   - Welcome: index.md
   - Core:
     - Overview: core/index.md
-    - Concepts:
-      - Mantra: core/concepts/mantra.md
-      - Getting Started:
-        - First Steps: core/concepts/getting-started/index.md
-        - Data Pipelines: core/concepts/getting-started/data-pipelines.md
-        - Table Definitions: core/concepts/getting-started/table-definitions.md
-        - Diagrams: core/concepts/getting-started/diagrams.md
-        - Dependencies: core/concepts/getting-started/dependencies.md
-      - Query Language:
-        - Data Types: core/concepts/query-lang/data-types.md
-        - Query Objects: core/concepts/query-lang/query-objs.md
-        - Operators:  core/concepts/query-lang/operators.md
-        - Common Commands: core/concepts/query-lang/common-commands.md
-        - Primary Key: core/concepts/query-lang/primary-key.md
-        - Normalization: core/concepts/query-lang/normalization.md
-      - Reproducibility:
-        - Table Tiers: core/concepts/reproduce/table-tiers.md
-        - Make Methods: core/concepts/reproduce/make-method.md
-      - Referential Integrity:
-        - Query Backend: core/concepts/ref-integrity/query-backend.md
-        - Distributed Computing: core/concepts/ref-integrity/distributed-computing.md
-        - Relational Databases: core/concepts/ref-integrity/relational-databases.md
-      - Glossary: core/glossary.md
-      - FAQ: core/faq.md
+    - Concepts: 
+      - Principles: core/concepts/principles.md
+      - Glossary: core/concepts/glossary.md
```

Comment:
Why not have the glossary to be further out? This seems a bit buried. For example, I normally review the glossary first to establish terms before reading deeper in the documentation of other projects.

Reply:
TBD in subsequent PR's.